grib2_to_cb/test/test_unit_proj.py

```diff
@@ -0,0 +1,120 @@
+# pylint: disable=missing-module-docstring
+import os
+import xarray as xr
+import pyproj
+from grib2_to_cb.run_ingest_threads import VXIngest
+
+def setup_connection():
+    """test setup"""
+    try:
+        _vx_ingest = VXIngest()
+        _vx_ingest.credentials_file = os.environ["CREDENTIALS"]
+        _vx_ingest.cb_credentials = _vx_ingest.get_credentials(_vx_ingest.load_spec)
+        _vx_ingest.connect_cb()
+        _vx_ingest.load_spec["ingest_document_ids"] = _vx_ingest.collection.get(
+            "JOB-TEST:V01:METAR:GRIB2:MODEL:HRRR"
+        ).content_as[dict]["ingest_document_ids"]
+        return _vx_ingest
+    except Exception as _e:  # pylint:disable=broad-except
+        assert False, f"test_credentials_and_load_spec Exception failure: {_e}"
+        return None
+
+def test_proj():
+    """test the proj"""
+    vx_ingest = None
+    try:
+        vx_ingest = setup_connection()
+        ds_height_above_ground_2m = xr.open_dataset(
+            "/opt/data/grib2_to_cb/hrrr_ops/input_files/2120013000018",
```

Comment:
I'm trying to recall if we had a solution to avoid hard-coding this data location that we could use here. That may have just been for our test output though.